test/helpers/client-stub.js

```diff
@@ -1,11 +1,9 @@
-const mentions = require('./mentions');
+const UserStub = require('./user-stub');
```

Comment:
Turns out it isn't too hard to create Fixtures and pass them into the constructor. Next commit will drop almost all the stubs (client might remain) and switch to official classes